### DISCUSSION TITLE
Why Slower than...

### DISCUSSION THREAD

**Discussion by jd1500** [SpiralLinux/SpiralLinux-project#33] [General]
As a new user I just HAVE to know...

Why "SpiralLinux:  Linux, faster than a **snai**l"??

Thanks for any replies!

**Comment by robertocannito** [comment]
Hi all depend from your hardware I think, on my two notebook (1 with intel I 7 16 gb ram ssd - 1 with ryzen 5 8 gb ram) run like a charme and without any problem

Installed Gnome DE in testing branch

Rob

**Reply by jd1500** [reply]
Thanks Rob, but I think you misunderstood my question?   It runs super fast on my old PC, and that is why I wondered why it was called "faster than a snail" ?

**Reply by geckolinux** [reply]
Hi there @jd1500 , I was wondering when somebody would ask that. ;-) It's just my lame attempt at playing with the concept of Debian's spiral logo and the SpiralLinux logo that is a reverse spiral design with elements of a snail shell design, and then combined with a bit of a parody on trendy marketing catchphrases. And after all, it _is_ faster than a snail isn't it? ;-)

**Reply by robertocannito** [reply]
:) :)

**Reply by jd1500** [reply]
**GeckoLinux,**  Thank you for that explanation...It defines the depths of your imagination!
